documentation/assemblies/cruise-control/assembly-cruise-control-concepts.adoc

```diff
@@ -5,17 +5,21 @@
 [id='cruise-control-concepts-{context}']
 = Using Cruise Control for cluster rebalancing
 
-include::../../modules/cruise-control/con-cruise-control-description.adoc[leveloffset=+1]
+[role="_abstract"]
+Cruise Control is an open-source system for Kafka that monitors broker loads and rebalances partitions to optimize use of resources across the cluster.
```

Comment:
I see we use the word  "system" in the original doc before these changes but is it the correct word? Wouldn't "application" be a better fit here?
```suggestion
Cruise Control is an open-source application designed to run alongside Kafka to help optimize use of cluster resources by:
* Monitoring cluster workload
* Rebalancing partitions based on predefined constraints
```